embedded_sass.proto

```diff
@@ -89,3 +89,86 @@ message CompileResponse {
   // generated CSS can be reached from the source map.
   string source_map = 3;
 }
+
+// An oubound event indicating that a message should be displayed to the user.
+message LogEvent {
+  // The request id for the compilation that triggered the message.
+  uint32 compilation_id = 1;
```

Comment:
```suggestion
  uint32 id = 1;
```

maybe? Since we used id for all the other fields

Reply:
I have this as `compilation_id` because I want it to be clear that this is *not* a response to `CompilationRequest`. Also, I expect there will be some outbound requests/inbound responses that are tied to a given compilation, so they might have both a `compilation_id` representing which compilation they're involved in *and* an `id` representing that specific request/response pair.

Reply:
Makes sense